### ISSUE TITLE
Performance Guide  : figures are not visible

### ISSUE THREAD

**Issue by didier-durand** [google/gvisor#4017]
Hi,

On my Mac with Firefox, I cannot see any of the charts published on this page : I just see white rectangles instead

Thanks!

Didier

**Comment by ianlewis** [comment]
Yeah, it may be a problem with the javascript on that page. We likely only tested it with chrome :(

**Comment by ianlewis** [comment]
I don't have a local firefox install so I loaded the page in headless mode and got the following errors.
```
$ firefox --profile rust_mozprofilezKFBAt/  --headless --screenshot https://gvisor.dev/docs/architecture_guide/performance/
*** You are running in headless mode.
JavaScript error: undefined, line 14: Error: An unexpected error occurred
JavaScript error: moz-extension://9655ddb8-bc17-4660-9da8-bd4bb522e6bf/lib/picture_in_picture_overrides.js, line 15: Error: Incorrect argument types for pictureInPictureParent.setOverrides.
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 1: ReferenceError: render_sysbenchmemory is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 536: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 761: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 974: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 1192: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 1406: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 1630: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 1858: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 2068: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 2293: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 2501: ReferenceError: d3 is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 1: ReferenceError: render_httpd100k is not defined
JavaScript error: https://gvisor.dev/docs/architecture_guide/performance/, line 1: ReferenceError: render_ffmpeg is not defined
Screenshot saved to: /usr/local/google/home/ianlewis/tmp/screenshot.png
JavaScript error: resource://gre/modules/AutoCompletePopup.jsm, line 113: NS_ERROR_ILLEGAL_VALUE: Component returned failure code: 0x80070057 (NS_ERROR_ILLEGAL_VALUE) [nsIObserverService.removeObserver]
```

**Comment by didier-durand** [comment]
You're right : I cross-checked with Chrome & Safari on my Mac. Both are fine. So, it's only a FF issue. 
Best,
Didier

**Comment by EtiennePerot** [comment]
This was fixed in 919910ce670a680ec9b9438c8087884b16385b03.
